_specifications/lsp/3.17/types/markupContent.md

```diff
@@ -102,4 +102,4 @@ Known markdown parsers used by clients right now are:
 Parser          | Version | Documentation
 --------------- | ------- | -------------
 marked          | 1.1.0   | [Marked Documentation](https://marked.js.org/)
-Python-Markdown | 3.2.2   | [Python-Markdown Documentation](https://python-markdown.github.io)
```

Comment:
python3 -m pip install SomePackage[PDF]
python3 -m pip install SomePackage[PDF]==3.0
python3 -m pip install -e .[PDF]  # editable project in current directory